### ISSUE TITLE
Arithmetic operator is not defined for arguments of types (xs:string, xs:string)

### ISSUE THREAD

**Issue by bermud** [opengeospatial/ets-wcs11#2]
When I try to run the WCS 1.1.1 test, most of the GetCoverage tests fail with
the error:
Arithmetic operator is not defined for arguments of types (xs:string,
xs:string)
I suspect there is a problem the variables used to calculate the boundingbox,
possibly the ones used in one of the following lines which are repeated in all
these test:
````   
    
<xsl:variable name=""LengthX"" select=""$UpperCornerX - $LowerCornerX""/>
<xsl:variable name=""LengthY"" select=""$UpperCornerY - $LowerCornerY""/>
<xsl:variable name=""DeltaX"" select=""$LengthX * 0.025""/>
<xsl:variable name=""DeltaY"" select=""$LengthY * 0.025""/>
````
                   
[Originally reported by Olsson, Mats in the OGC CITE issue tracker: issue #954](https://portal.opengeospatial.org/services/srv_public_issues.php?call=viewIssue&issue_id=954)  

_Opened: 2014-07-15 10:06:09 Last Updated: 2014-07-15 10:17:29_

**Comment by dbretschneider** [comment]
I just added the lines from Luis from https://github.com/opengeospatial/ets-wcs11/issues/2#issuecomment-50082655.

See commit:
 * https://github.com/lat-lon/ets-wcs11/commit/4d5461bd50b4eea79cc624c68df17400e8ab95e6

Is a WCS 1.1.1 service available for testing?

**Comment by dstenger** [comment]
Can you provide a service URL which can be used to test the fix?

**Comment by matsoo** [comment]
I have tested on the beta site with the new 1.6 version of the tests and this still happens for some of the coverage tests. I have not checked if the cause for the problem is the same. We have been unable to run the wcs 1.1.1 tests at all for some time because of a schema issue.

You can use the server at:
http://192.71.92.21/MapService/wcs.axd/Dem? 
if you want to test.

wcs:GetCoverage_EmptyIdentifier, wcs:GetCoverage_BogusIdentifier fails with the same error message in 1.6. wcs:GetCoverage_MissingBoundingBox and wcs:GetCoverage_MissingIdentifier throws an exception from another issue so I cannot tell if they also are affected.

**Comment by dstenger** [comment]
We have implemented a fix and tested it successfully with your provided URL.

A pull request for this fix was created: #14

The fix will be part of the next release.

**Comment by dstenger** [comment]
Pull request was merged successfully into master branch and is part of the upcoming release.
